src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -346,8 +347,8 @@ private void FinishSolutionBindingOnUIThread(IProgressController controller, Can
                     // TODO: SVS-33 (https://jira.sonarsource.com/browse/SVS-33) Trigger a Team Explorer warning notification to investigate the partial binding in the output window.
                     this.AllNuGetPackagesInstalled &= isNugetInstallSuccessful;
 
-                    progressNotifier.NotifyIncrementedProgress(string.Empty);
```

Comment:
Properly fixing the progressNotifier creation (see my previous comment) should be enough. We don't need to move this line.